src/Engine/Entity/Entity.hpp

```diff
@@ -58,9 +59,10 @@ class Entity {
         /// Instantiate a scene as a child to this entity.
         /**
          * @param name The name of the scene to instantiate.
+         * @param isSameScene set to true if it is the same scene you want to istantiate.
```

Comment:
Change "isSameScene" name to example: "copy",  would be more descriptive of what it actually does.